datafusion/physical-plan/src/windows/mod.rs

```diff
@@ -112,25 +112,9 @@ pub fn create_window_expr(
             ))
         }
         WindowFunctionDefinition::AggregateUDF(fun) => {
-            // Convert `Vec<PhysicalSortExpr>` into `Vec<Expr::Sort>`
-            let sort_exprs = order_by
-                .iter()
-                .map(|PhysicalSortExpr { expr, options }| {
-                    let field_name = expr.to_string();
-                    let field_name = field_name.split('@').next().unwrap_or(&field_name);
-                    Expr::Sort(SortExpr {
-                        expr: Box::new(col(field_name)),
-                        asc: !options.descending,
-                        nulls_first: options.nulls_first,
-                    })
-                })
-                .collect::<Vec<_>>();
-
             let aggregate = AggregateExprBuilder::new(Arc::clone(fun), args.to_vec())
                 .schema(Arc::new(input_schema.clone()))
                 .alias(name)
-                .order_by(order_by.to_vec())
-                .sort_exprs(sort_exprs)
```

Comment:
`sort_exprs` is removed. I think we could take a look again with the latest main branch

Reply:
Thank you. I rebased on `main` and ended up getting a different error, but I still think we need a correction. I think I'll write up a unit test as well.

Reply:
I guess after we introduce WindowExprBuilder, we could have a nice builder for createing window aggregate function with single `order_by`.